odoo/addons/base/models/ir_actions_report.py

```diff
@@ -741,27 +741,32 @@ def close_streams(streams):
                                 break
                             node = node['/Next']
```

Comment:
maybe we could have:

```py
outlines_pages = []
if '/Outlines' in root and '/First' in root['/Outlines']:
    node = root['/Outlines']['/First']
    while True:
        outlines_pages.append(root['/Dests'][node['/Dest']][0])
        if '/Next' not in node:
            break
        node = node['/Next']
    outlines_pages = sorted(set(outlines_pages))
if len(outlines_pages) == len(res_ids) and outlines_pages[0] == 0:
```
and not duplicate the else (in this branch `len(res_ids) > 1`)